test/helpers_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+describe Lotus::Assets::Helpers do
+
+  def safe_string_class
+    ::Lotus::Utils::Escape::SafeString
+  end
+
+  describe '#javascript' do
+    before do
+      @javascript = View.new.javascript('feature-a')
+    end
+
+    it 'returns an instance of SafeString' do
+      @javascript.must_be_instance_of safe_string_class
```

Comment:
Uhmmm .... I think the other tests do that. 

Lotus is escaping the output string. This gem was OK until lotus 0.4.x was released. To write HTML tags on lotus we only need to return an instance of `SafeString`.

What do you think?